app/models/comment.rb

```diff
@@ -328,6 +332,22 @@ def self.gmail_parsed_mail(mail_doc)
     }
   end
 
+  def self.outlook_parsed_mail(mail_doc)
+    if mail_doc.css(".outlook_quote").any?
```

Comment:
There is nothing like `outlook_quote` class in outlook email. And the implementation for reply by email from outlook is different from that of the Gmail and Yahoo. So please don't copy the Gmail and Yahoo mail implementation. Instead, you have to analyze the emails sent by outlook to Gmail and outlook to other emails by converting them to HTML and them extract out how we can separate out the quoted element of the mail and actual reply.
This is a harder issue. It is gonna take some time researching.
Thanks

Reply:
> Instead, you have to analyze the emails sent by outlook to Gmail and outlook to other emails by converting them to HTML and them extract out how we can separate out the quoted element of the mail and actual reply.

Can you pls explain me this.. I really want to solve this issue...